lib/routes/user/home/account_page.dart

```diff
@@ -49,6 +49,8 @@ class _AccountPageState extends State<_AccountPage> {
   StreamSubscription<AccountModel> _statusSubscription;
   String _filter;  
   String _paymentRequestInProgress;
+  PaymentFilterModel _paymentFilterModel;
```

Comment:
don't save this value in the class, use it locally in the stream in the build function.